mojap_airflow_tools/operators.py

```diff
@@ -116,6 +123,24 @@ def basic_kubernetes_pod_operator(
         if k not in env_vars:
             env_vars[k] = v
 
+    if user_name is None:
+        security_context = (
+            {
+                "allowPrivilegeEscalation": False,
+                "runAsNonRoot": True,
+                "privileged": False,
+            },
+        )
+    else:
+        security_context = (
+            {
+                "allowPrivilegeEscalation": False,
+                "runAsNonRoot": True,
+                "runAsUser": user_name,
+                "privileged": False,
+            },
+        )
```

Comment:
You could make this more concise:
```suggestion
    security_context = {
        "allowPrivilegeEscalation": False,
        "runAsNonRoot": True,
        "privileged": False,
    }
    if user_name is not None:
        security_context["runAsUser"] = user_name
```